.vscode/settings.json

```diff
@@ -1,17 +1,14 @@
 {
+  "editor.guides.bracketPairs": "active",
```

Comment:
Only if it's a change we want enabled by default for our team. It should probably go in user settings instead.

Unfortunately VSCode doesn't have support for shared workspace settings and custom workspace settings. It's just workspace and global. There's a longstanding issue to support it https://github.com/microsoft/vscode/issues/40233

Basically some teams don't commit `.vscode/settings.json` because the devs want all settings within their vscode separated by project. And some do because they have a default list of settings that is useful (like ours)